doc/api/zlib.md

```diff
@@ -397,6 +397,9 @@ added: v0.5.8
 Not exported by the `zlib` module. It is documented here because it is the base
 class of the compressor/decompressor classes.
 
+This class inherits from [`stream.Transform`][], allowing `zlib` objects to be
```

Comment:
For `` [`stream.Transform`][] `` to be turned into a link, you need to add this line in the bottom of the file, among other references:

```md
[`stream.Transform`]: stream.html#stream_class_stream_transform
```

As the bottom references are sorted in ASCII order, add this line before this one:
```md
[`zlib.bytesWritten`]: #zlib_zlib_byteswritten
```

Reply:
As for links in Markdown, see https://daringfireball.net/projects/markdown/syntax#link

Reply:
Thanks. Reference-style links were new to me.